### PR TITLE
Bloodmallets memes blogpost

### PR DESCRIPTION
**Pull request by Bloodmallet**

### CODE REVIEW

**Review by HawkCorrigan — CHANGES_REQUESTED** [review]
Rest looks good ^^

File: blog/_posts/2022-02-15-up-and-coming-memes.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+**Stats**: haste >= crit > vers > mastery
+
+
+### The Chainer
+> Kyrian/Venthyr & Chains of Devastation
+
+The Shadowlands posed quite a few surprises for us. 
+During the time we tried to recover our companions from Torghast we freed many tormented souls from their chains.
+It's time to pick them up and bring the fight to the Jailor, as the Chainer.
```
Comment (Eokira24601) [inline blog/_posts/2022-02-15-up-and-coming-memes.md]: Jailer*

File: blog/_posts/2022-02-15-up-and-coming-memes.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+title: Up and coming memes
+date: 2022-02-04 00:00:00 +0200
+excerpt: 9.2 is bringing more than Tier and double Legendaries
+author: Bloodmallet(EU)
+toc: true
+---
+Greetings,
+
+## Community questions 
+1. will the return of Tier sets be exciting or trash for mythic plus?
```
[the remark this answers is not in the document]
Comment (Bloodmallet) [reply blog/_posts/2022-02-15-up-and-coming-memes.md]: Fixed

**Review by Eokira24601 — COMMENTED** [review]
Looks good :)

File: blog/_posts/2022-02-15-up-and-coming-memes.md (hunk 1 above)
Comment (Bloodmallet) [reply blog/_posts/2022-02-15-up-and-coming-memes.md]: fixed

**Pull request merged**